data_structures/heap/merge_kSorted_Arrays.py

```diff
@@ -0,0 +1,37 @@
+from queue import PriorityQueue
```

Comment:
Delete this file and move it to a separate pull request because we only want one algorithm per pull request.